### PR TITLE
#160056365 setup basic react app

### PR DESCRIPTION
**Pull request by tutugodfrey**
#### What does this PR do?
Install dependencies and setup a react app

#### Description of Task to be completed?
- Install dependencies
- configure webpack for bundling
- add index.html
- create App component
 
#### How should this be manually tested?
- clone the repository and run `npm install` to install dependencies
- run `npm run dev` to serve the app

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#160056365

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "presets": [
+        "env", "react"
+    ]
+}
```
Comment (rhenshaw56) [inline .babelrc]: EOL

**Review by rhenshaw56 — APPROVED** [review]
Please fix the hound comments

File: public/styles/App.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+} 
+
+
+body{
+    color: $body-color;
+    background-color: $body-background;
+    background-image: url('../assets/beverage-books.jpg') ;
+    background-size: 100% auto;
+    background-repeat: no-repeat;
+}
```
Comment (rhenshaw56) [inline public/styles/App.scss]: EOL

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+	"presets": [
+		"env", "react"
```
Comment (rhenshaw56) [inline .babelrc]: what about the presets for stage-3 ?

File: .gitignore [hunk 4]
```diff
@@ -1 +1,2 @@
-node_modules/
+node_modules
+.env
```
Comment (rhenshaw56) [inline .gitignore]: we won't be needing an env file so get rid of this

File: .hound.yml [hunk 5]
```diff
@@ -2,3 +2,6 @@ eslint:
   enabled: true
   config_file: 
     .eslintrc.json
+
+scss:
+  enabled: false
```
Comment (rhenshaw56) [inline .hound.yml]: EOL

**Review by rhenshaw56 — CHANGES_REQUESTED** [review]
I stopped reviewing this PR when I saw the your build files here, can you tell me why it's here ?

File: package.json [hunk 6]
```diff
@@ -16,11 +19,36 @@
     "url": "https://github.com/andela/rivendell-ah-client/issues"
   },
   "homepage": "https://github.com/andela/rivendell-ah-client#readme",
+  "dependencies": {
+    "dotenv": "^6.0.0",
```
Comment (rhenshaw56) [inline package.json]: remove dotenv, you don't need it

File: public/styles/App.scss [hunk 7]
```diff
@@ -0,0 +1,14 @@
+$body-color : green;
+$body-background: rgb(218, 216, 216);
+
+.primary {
+	background-color: red;
```
Comment (rhenshaw56) [inline public/styles/App.scss]: can you use rgb() style instead of just inputing the colour names

File: webpack.config.js [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  const common = {
+    entry: path.join(__dirname, 'src', 'index.js'),
+    output: {
+      path: path.join(__dirname, './dist'),
+      filename: 'bundle.js',
+    },
+    module: {
+      rules: [
+        {
+          test: /.js$/,
```
Comment (rhenshaw56) [inline webpack.config.js]: add a test for .jsx also

File: webpack.config.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
+const path = require('path');
```
Comment (rhenshaw56) [inline webpack.config.js:1]: add webpack config to eslintrc

File: src/index.js [hunk 10]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (rhenshaw56) [reply src/index.js:1]: @tutugodfrey @chidioguejiofor attend to this

**Review by rhenshaw56 — CHANGES_REQUESTED** [review]

**Review by megame24 — APPROVED** [review]

**Review by rhenshaw56 — APPROVED** [review]

**Pull request merged**